docs/multicluster.md

```diff
@@ -288,3 +288,54 @@ First, per the [article](https://docs.microsoft.com/en-us/azure/aks/update-crede
 
 Then follow the steps for [updating a deployment parameter](#updating-a-deployment-parameter) above.
 
+## Updating Cluster Node SSH Keys
+
+When setting up the AKS cluster, one specifies the public component of an SSH key using the variable `ssh_public_key`.  This value is used to configure each of the nodes within the AKS cluster for SSH key authentication to log into the cluster nodes (using the private key).  Periodically, there will be a need to rotate these keys for security or other purposes.
+
+Currently, there are two methods for updating the SSH keys:
+
+- Update the `ssh_public_key` value and using Terraform
+- Manually updating the keys using the Azure CLI
+
+As of the writing of this document, each method has tradeoffs.
+
+### Update SSH Key Using Terraform
+
+Updating the SSH key using Terraform is not much different than updating other values like the Service Principal (above).  However, updating the SSH key using Terraform will result in the cluster being replaced by a new cluster rather than updating the current cluster in place.  To update the SSH key, one would modify your `tfvar` file by updating the `ssh_public_key`, say from:
```

Comment:
"To update the SSH key, one would modify the `tfvar` file by...". Maybe stick to either third person or second person?